.github/pull_request_template.md

```diff
@@ -0,0 +1,55 @@
+## Title (delete after following this rule)
+
+Your PR title should be prefixed with the Jira ticket ID, e.g. "ADX-123 very important feature".
+
+## Description
+
+Please include a summary of the change(s).
+You should also include motivation: why was this change required and what problem it solves.
+
+Don't forget to reference all other pull requests which are associated to this one below, especially if merging need to be coordinated.
+E.g.
+
+relates fjelltopp/repository_name#issue_number
+
+## Dependency Changes (delete if not applicable)
+
+List any dependencies that have been removed/added/changed in this pull request.
+Has the pipfile.lock or yarn.lock been updated?
```

Comment:
Dependency changes will most often require a PR to central project repo (adx_deploy for ADR or spc for Spectrum CKAN). I think this part should be referencing this PR but also explain why you're adding/modifying depencies.